civihr_default_theme/scss/base/_custom.scss

```diff
@@ -386,7 +370,12 @@ a {
 
 /* Headings */
 
-h1 {
+.block__title {
```

Comment:
I've never seen Drupal using the BEM convention, nor I could find this class in the actual page